src/components/Comments/ReplyCommentItem.js

```diff
@@ -190,7 +206,7 @@ const ReplyCommentItem = ({
   const onPressDMAnon = async () => {
     try {
       setLoading({...loading, loadingDmAnon: true});
-      await sendMessageDM(comment?.id, 'comment', 'ANONYMOUS');
+      await sendMessageDM(comment?.id, 'comment', 'ANONYMOUS', feedId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, ensure that the addition of `feedId` in the `sendMessageDM` call in the `onPressDMAnon` function doesn't affect existing functionality.



<!-- This is an auto-generated comment by OSS CodeRabbit -->